homeassistant/components/system_bridge/manifest.json

```diff
@@ -3,11 +3,11 @@
   "name": "System Bridge",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/system_bridge",
-  "requirements": ["systembridge==2.3.1"],
+  "requirements": ["systembridgeconnector==3.0.1"],
```

Comment:
All the requirements required for the production install of the `systembridgeconnector` package are pinned to exact versions. We don't want that to avoid conflicts with other integrations with libraries that require the same requirements. Please be as forgiving as possible when pinning version specifiers in the library.

https://github.com/timmo001/system-bridge/blob/084eb233d6c1293979ffbf66512f2a8b2f2d7ecc/connector/setup.py#L6-L24

https://github.com/timmo001/system-bridge/blob/084eb233d6c1293979ffbf66512f2a8b2f2d7ecc/connector/requirements.txt#L1-L4

Normally `requirements.txt` with exact pinning is used for development and tests and a separate set of requirements without pinning or with loose pinning are specified for `setup.py` for production.